guide/login.md

```diff
@@ -0,0 +1,16 @@
+## User Handbook
+### Logging In
+
+Once you have [created a user account](https://github.com/Arkinn/docs/blob/testing/guide/registration.md) the next step to interacting with the community will be to login.
+
+The login screen can be accessed by clicking **Log In** at the top right of the screen, the animation below will show you what this looks like:
+
+![Animation - Login page](687474703a2f2f692e696d6775722e636f6d2f616c56484e53702e676966.gif)
+
+Enter your **Username** _OR_ **e-mail address** and **Password** that you filled out during the [sign up process](https://github.com/Arkinn/docs/blob/testing/guide/registration.md) and then click Log In. 
```

Comment:
Some communities will have SSO extensions (Facebook, Twitter, Github, and others) configured, and if the user registered through those they will **have** to log in via those buttons (unless they [migrate to a regular login method](https://discuss.flarum.org/d/5744-how-to-migrate-an-o-auth-account-to-regular-account)). This should be part of the documentation because those users won't have passwords to enter.